example/hla/SomeModule.module

```diff
@@ -16,7 +16,7 @@ ValueObjects
 
     SomeClass
         id: SomeId
-        amount: int
+        amount: int (startsFrom: 10)
```

Comment:
add it also to some simple VO to check if it works

Reply:
added and also fixed, thx for noticing